website/client/assets/scss/task.scss

```diff
@@ -243,7 +243,7 @@
     &-control {
       &-bg { 
         background: rgba(255, 217, 160, 0.32) !important;
-        .small-text { color: $yellow-10 !important; }
+        .small-text { color: #EE9109 !important; }
```

Comment:
This is in colors.scss as `$orange-10` (https://github.com/HabitRPG/habitica/blob/v4.65.5/website/client/assets/scss/colors.scss#L63), so you can use that instead of the raw hex value.

Reply:
It looks like we have two instances of `$orange-10` is that a concern?